packages/samples/headless-react/src/components/breadcrumb-manager/breadcrumb-manager.class.tsx

```diff
@@ -0,0 +1,145 @@
+import {Component, ContextType} from 'react';
+import {
+  buildBreadcrumbManager,
+  BreadcrumbManager as HeadlessBreadcrumbManager,
+  BreadcrumbManagerState,
+  Unsubscribe,
+  NumericFacetValue,
+  DateFacetValue,
+} from '@coveo/headless';
+import {AppContext} from '../../context/engine';
+import {parseDate} from '../date-facet/date-utils';
+
+export interface BreadcrumbManagerProps {
+  numericFormat: (n: number) => string;
```

Comment:
I guess for the example sake, we could keep the `numericFormat` as just an internal thing/not a prop on the component ?